Mono.Cecil/ModuleKind.cs

```diff
@@ -32,6 +32,7 @@ public enum TargetArchitecture {
 		ARM = 0x01c0,
 		ARMv7 = 0x01c4,
 		ARM64 = 0xaa64,
+		RiscV64 = 0x5064,
```

Comment:
Maybe add LA64 while touching it:
```suggestion
		RiscV64 = 0x5064,
		LoongArch64 = 0x6264,
```

Reply:
Thank you. I will add AnyCPU for RiscV64 and LoongArch64 as jkotas reviewed.